### DISCUSSION TITLE
Lock file chnages version in production

### DISCUSSION THREAD

**Discussion by Arctomachine** [pnpm/pnpm#7277] [Q&A]
In one of projects lock file constantly changes version on production server, which makes me to `git restore` each time I push new version
```sh
# git diff
diff --git a/pnpm-lock.yaml b/pnpm-lock.yaml
index 760a92a..d8c27e1 100644
--- a/pnpm-lock.yaml
+++ b/pnpm-lock.yaml
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
It happens only to lock file in monorepo root, which is 6.0. Lock files in packages are 5.4 and are not changed. What am I doing wrong and how to fix it?

Dependencies in root package.json have not changed in last year

**Comment by pivstone** [comment]
https://github.com/pnpm/pnpm/issues/6609

you may need to lock pnpm version or upgrade all the pnpm version to a newer one

**Reply by zkochan** [reply]
This is the correct answer. We have changed the version of the lockfile back to 6.0 in order to prevent compatibility issues. If you upgrade pnpm, the lockfile version will be always 6.0.
